docs/concepts/logging.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Logging
+weight: 10
+---
+
+## Logging
+
+Lokomotive restricts the logs per container to be 300MB divided in three files (100MB per file).
```

Comment:
Please justify the numbers, associated assumptions and explain the limitations of this configuration.

Reply:
> Lokomotive restricts the logs per container to be 300MB divided in three files (100MB per file). Due to this limitation any log intensive app will have only last 300MB of logs anything beyond that will be purged by Docker. To preserve logs of such applications for long term access use a log shipping solution.

How about ^ this?

Reply:
I would add, that this should allow to run around 50 logging intensive containers on 20GB disk (Flatcar takes a bit of space, but we calculated limit of 66, so saying 50 sounds reasonable I think). And I would note, this is a Docker _containers_ and not pods, as one pod consist of at least 2 containers.

Reply:
> Lokomotive restricts the logs per _Docker container_ (not _Kubernetes pods_) to be 300MB divided into three files (100MB per file). This should allow running around fifty log intensive containers on a 20GB disk. Due to this limitation, any log intensive app will have only last 300MB of logs anything beyond that will be purged by Docker. To preserve logs of such applications for long term access use a log shipping solution.

Reply:
Sounds great :+1: Thanks!